src/main/java/edu/unc/lib/boxc/migration/cdm/services/FindingAidService.java

```diff
@@ -51,11 +56,11 @@ public void recordFindingAid() throws Exception {
             if (key.equalsIgnoreCase(CONTRI_FIELD) && value.equalsIgnoreCase(HOOK_ID)) {
                 project.getProjectProperties().setHookId(key);
                 ProjectPropertiesSerialization.write(project);
-                outputLogger.info("HookId was set. Use 'config list' to view the project property.");
+                outputLogger.info(HOOK_ID + " was set. Use 'config list' to view the project property.");
```

Comment:
Sorry, I meant use `ProjectPropertiesService.HOOK_ID`, since that is the name of the property the user would have to change using the config command if they disagreed with the automatic assignment that took place. Its a little different from the constant from this file (which is the description of the field itself). It might be less confusing if you renamed the constants declared in this class to `HOOK_ID_FIELD_DESC` and `COLLECTION_NUMBER_FIELD_DESC`